### ISSUE TITLE
Signup needs more fields

### ISSUE THREAD

**Issue by weirdwiz** [pranshugrover/Forms#5]
The signup page requires more fields, namely
- Name
- Email
- Confirm Password

We should also validate and sanitise inputs before putting them into the database

**Comment by Ankurk99** [comment]
I don't think we need Name and e-mail field. Username(distinct) and password are enough to signup.
And also isn't making a form which doesn't require any email was our intention to make this form "Open"?
